lib/pdk/cli/validate.rb

```diff
@@ -15,6 +15,7 @@ module PDK::CLI
 
     flag nil, :list, _('list all available validators')
     flag :a, 'auto-correct', _('automatically correct problems (where possible)')
+    flag nil, :parallel, _('run validations in parallel')
```

Comment:
Not necessary to comment it out, can just make it a hidden flag (like --answer-file), so that it's still available for dev & testing but not shown in the help output.